extensions/vercast/package.json

```diff
@@ -1,54 +1,55 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "vercast",
-  "title": "Vercel Deployments",
-  "description": "View your Vercel deployments",
+  "title": "Vercast",
+  "description": "View and modify your Vercel projects and deployments.",
   "icon": "icon.png",
-  "author": "matt",
+  "author": "MaxLeiter",
```

Comment:
I wonder if you can have multiple authors

Reply:
The schema specifically mentions `User currently contributing the most to the extension.`, so I don't think you can yet. It definitely should be updated to include @gleich when possible, though.

Reply:
You can only have one author, but you can add others users to `contributors` property